docs/manual/en/introduction/Creating-a-scene.html

```diff
@@ -37,6 +37,10 @@ <h2>Before we start</h2>
 		</code>
 
 		<p>That's all. All the code below goes into the empty &lt;script&gt; tag.</p>
+		
+		<h2>Using NPM</h2>
+		
+		<code>$ npm install three --save-dev</code>
```

Comment:
I'd omit `--save-dev`, or use just `--save` here — typically `three` will not be a dev dependency.